VideoLocker/src/main/java/org/edx/mobile/model/api/CourseEntry.java

```diff
@@ -55,7 +57,7 @@ public void setStart(String start) {
     }
 
     public String getCourse_image(Config config) {
-        return config.getApiHostURL() + course_image;
```

Comment:
Since null can now be returned, what happens when this returns null to the callers?